crates/db_schema/src/impls/comment.rs

```diff
@@ -236,6 +236,23 @@ impl Comment {
     }
     Ok(())
   }
+
+  pub async fn read_ap_ids_for_post(
```

Comment:
This would probably be better in the post impl, and rename it `read_comment_ap_ids`